src/modules/accommodations/services/accommodations.service.ts

```diff
@@ -100,4 +100,21 @@ export class AccommodationService {
     }
     return accommodation;
   }
+
+  async softDeleteAccommodationByOwner(
+    accommodationId: string,
+    ownerId: string,
+  ): Promise<void> {
+    const accommodation = await this.accommodationRepository.findOne({
+      where: { id: accommodationId, owner: { id: ownerId }, deletedAt: null },
```

Comment:
Could you please in the list/get accomodation endpoints add the clause to select those records that are not soft deleted?